solana/gatekeeper-lib/src/util/account.ts

```diff
@@ -1,21 +1,22 @@
 import { Keypair, Connection, PublicKey, clusterApiUrl } from "@solana/web3.js";
+// eslint-disable-next-line unicorn/prefer-node-protocol
```

Comment:
Please fix and don't ignore

Reply:
When I switch it to `node:fs`, the command at the bottom of the file `fs.readFileSync(file).toString("utf-8")` fails linting because fs is viewed as an "any" type so it flags no-unsafe-any